Snakefile

```diff
@@ -3,9 +3,9 @@ import shutil
 import re
 import csv
 import Bio.SeqIO
-import Bio.Alphabet
 import subprocess
 from snakemake.utils import min_version
+from Bio.Seq import Seq
```

Comment:
Is this needed?

Reply:
I had an error when doing: `record.seq = Seq(re.sub('X',"",str(record.seq)))` last time executing and this import solved it. However I did not have this error initially...

Reply:
Ok, sounds good. Maybe you can make the import consistent with other imports (`import Bio.SeqIO` instead of `from Bio.Seq import Seq`) and use `Bio.SeqIO(re.sub(...`.